pulumi/python/kubernetes/applications/sirius/__main__.py

```diff
@@ -378,6 +385,11 @@ def add_namespace(obj):
     pulumi.export('ipaddress', lb_ingress_ip)
     #pulumi.export('application_url', f'https://{lb_ingress_hostname}')
     application_url = sirius_host.apply(lambda host: f'https://{host}')
+elif infra_type == 'DO':
+    pulumi.export('hostname', lb_ingress_hostname)
+    pulumi.export('ipaddress', lb_ingress_ip)
```

Comment:
My mind desperately wants an underscore between ip and address.

Reply:
Mine too....but when I started digging I found out that it's "ipaddress" in all the library code so I decided to be consistent with the rest of the code.